### PR TITLE
Add ignored version state to bundle

### PR DESCRIPTION
**Pull request by TerryHowe**

### CODE REVIEW

File: api/v1alpha1/packagebundle_types.go [hunk 1]
```diff
@@ -113,14 +113,15 @@ type PackageBundleStatus struct {
...
 
-//+kubebuilder:validation:Enum={"inactive","active","active (upgrade available)","ignored"}
+//+kubebuilder:validation:Enum={"inactive","active","active (upgrade available)","ignored","ignored version"}
 type PackageBundleStateEnum string
 
 const (
 	PackageBundleStateInactive         PackageBundleStateEnum = "inactive"
 	PackageBundleStateActive           PackageBundleStateEnum = "active"
 	PackageBundleStateUpgradeAvailable PackageBundleStateEnum = "active (upgrade available)"
 	PackageBundleStateIgnored          PackageBundleStateEnum = "ignored"
```
Comment (ivyostosh) [inline api/v1alpha1/packagebundle_types.go:123]: I find "ignored" and "ignored version" not mutually exclusive by description. Is there a better name convention that can help address this issue?

Comment (TerryHowe) [reply api/v1alpha1/packagebundle_types.go:123]: Yeh, I'm not a big fan of the name either. I don't like state names with spaces in there. I don't like "active (upgrade available)", but that is where we went for for whatever reason. I'd kind of sooner PackageBundleStateUpgradeAvailable didn't exist just PackageBundleStateActive with a NOTE "upgrade available"

Comment (ivyostosh) [reply api/v1alpha1/packagebundle_types.go:123]: I hear what you are saying. Not for this PR, but I think if we are having doubt about those names, we should change it prior to GA. It seems to be better to address it early on before we accumulate a bigger user base.

**Pull request merged**